src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -49,8 +49,8 @@ public function __construct($queryBuilder, $manager = null, $class = null)
         }
 
         if ($queryBuilder instanceof \Closure) {
-            if (!$manager instanceof EntityManager) {
-                throw new UnexpectedTypeException($manager, 'Doctrine\ORM\EntityManager');
+            if (!$manager instanceof ObjectManager) {
+                throw new UnexpectedTypeException($manager, 'Doctrine\Common\Persistence\ObjectManager');
```

Comment:
a better solution would be to check against both EntityManager and EntityManagerInterface (or just EntityManagerInterface if we don't care about Doctrine 2.3). This code expects an ORM manager